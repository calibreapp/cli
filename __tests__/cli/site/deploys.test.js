import {
  runCLI,
  setupIntegrationServer,
  teardownIntegrationServer
} from '../../utils'

import listDeploys from '../../fixtures/listDeploys.json'

describe('site deploys', () => {
  beforeAll(async () => await setupIntegrationServer(listDeploys))
  afterAll(async () => await teardownIntegrationServer())

  test('lists all deploys', async () => {
    const out = await runCLI({ args: 'site deploys --site=test' })
    expect(out).toMatchSnapshot()
  })
})
