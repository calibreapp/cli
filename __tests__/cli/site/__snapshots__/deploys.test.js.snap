// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`site deploys lists all deploys 1`] = `
"3 deploys
UUID                                 | REVISION | REPOSITORY                        | USERNAME | CREATED            
8c68b9f6-b5bf-49bc-abd7-896db6338818 | v1       | https://github.com/calibreapp/app | Deployer | 12:00AM 17-Jan-2019
640d2460-13a7-417e-8478-08b1bf166fd1 | v2       | https://github.com/calibreapp/app | Deployer | 12:00AM 19-Jan-2019
db846ca0-a26c-4667-b971-b6710506d866 | v3       | https://github.com/calibreapp/app | Deployer | 4:00PM 22-Jan-2019 
To see deploys after v3, run: calibre site deploys --site=calibre --cursor=MjU=="
`;
